pkg/Apps/GraphMaker/graph.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * @license
+ * Copyright (c) 2022 Google LLC All rights reserved.
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file.
+ */
+
+export const graph = 
+{
+  "$meta": {
+    "name": "unripe-annual",
+    "id": "2577-5784-1061"
+  },
+  "nodes": {
+    "NodeCatalogNode1": {
+      "id": "NodeCatalogNode1",
+      "type": "NodeCatalogNode",
+      "displayName": "Node Catalog",
+      "connections": {
+        "nodeTypes": [
+          "nodeTypes",
+          "NodeInspectorNode1undefinedinspectorData"
+        ],
+        "categories": [
+          "categories",
+          "NodeInspectorNode1undefinedinspectorData"
+        ],
+        "newNodeInfos": [
+          "NodeInspectorNode1undefinedinspectorData"
```

Comment:
`NodeInspectorNode1undefinedinspectorData`
the `undefined` here i think is a bug...

Reply:
Yes, this is output from Graph!

Reply:
unfortunately, i couldn't make GraphMaker work for me, I'm getting a split panel, all empty... we should look at it together (probably in the new year)

Reply:
GraphMaker (Apps/GraphMaker) is not Graph! (Apps/Graph). GraphMaker is just a façade right now, as we make it possible to describe GraphMaker itself in terms of graphs.

Reply:
Sorry, yes, GraphMaker is not rendering properly, I probably just broke it, I will check.

Reply:
I changed all the containers to have 'container' in their slot names, as a simple way to filter the NodeTree. I failed to update `graph.js` for the new names, I believe this is why the GraphMaker looks weird.